styleguide/components/01-mixins/breakpoints/_breakpoints.scss

```diff
@@ -35,3 +45,23 @@
     @content;
   }
 }
+
+//
+// Desktop-only breakpoint (see $bp-desktop defined in _vars.scss).
+// This asumes a min-width as defined by the $bp-desktop variable and a max-width as defined by the $bp-desktop-xl variable.
+//
+@mixin desktop-only {
```

Comment:
Not a fan of this approach, it's beter to avoid this use case.

Reply:
Agreed, we'll reomve the mixin-only() mixins.